vip-decoupled.php

```diff
@@ -43,6 +43,13 @@
 	require_once __DIR__ . '/blocks/blocks.php';
 }
 
+/**
+ * VIP Block Data API.
+ */
+if ( is_plugin_enabled( 'block-data-api' ) ) {
```

Comment:
Is there a chance the Block Data API also lands in `mu-plugins`? If so, and you want to prefer the mu-plugins version, you might want to proactively add some logic here to determine which version "wins."

Reply:
Good question! We have that logic built-in. In the plugin itself [we verify that we're not double-loading](https://github.com/Automattic/vip-decoupled-bundle/blob/7a8401c0146699e18ac2412bfdd268f5e0a2d520/lib/vip-block-data-api/vip-block-data-api.php#L20). On the `mu-plugins` integration side, we also have logic to [prefer the customer version of the plugin](https://github.com/Automattic/vip-go-mu-plugins/blob/d0af967/integrations/block-data-api.php#L39), if installed.

I spent some time just now testing what happens if the Block Data API plugin is installed, as well as the bundle with the Block Data API feature enabled, as well as the `mu-plugins` version. The priority goes:

1. Plugin version first
2. Bundle version second, if the plugin is disabled (second alphabetically)
3. The `mu-plugins` version of the plugin, if everything else is disabled

This fits our original intent to prefer customer code first, and works fine between the bundle and normal Block Data API plugin as well. Everything is working as designed.